Templates/users/users-edit.pug

```diff
@@ -0,0 +1,62 @@
+extends ../layout
```

Comment:
this file is VERY similar to `users-add.pug` can we not combine them?

Reply:
since users-edit.pug uses pre-filled fields and users-add.pug uses empty fields, I will need to add conditions in a lot of places to differentiate between edit and add. I think am missing something here, but I can fix this in the next PR if we have to